.vscode/settings.json

```diff
@@ -8,7 +8,8 @@
     "gopls": {
         "build.directoryFilters": [
             "-sdk",
-            "-examples"
+            "-examples",
+            "-templates",
```

Comment:
Err...I assume VSCode is OK with the trailing comma here?

Reply:
Normally it should be fine, but removed it just in case, thanks!